packages/@ember/-internals/glimmer/lib/glimmer-component-docs.ts

```diff
@@ -383,3 +383,5 @@
   @module @glimmer/component
   @public
 */
+
+export {};
```

Comment:
This is an empty file that exists to produce documentation. Without any imports or exports typescript worries that it's a non-module script, and that would be an error under isolatedModules.